content/decisions-with-conditionals/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Chapter 5: Making Decisions with Conditionals"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 5
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
```

Comment:
Add details to Learning Objectives and Key Terminology

Reply:
added learning objectives and key terms